.github/workflows/check-text.yml

```diff
@@ -0,0 +1,16 @@
+name: Check text linter rules # cSpell:ignore refcache startswith
+
+on:
+  pull_request:
+
+jobs:
+  check-formatting:
+    name: Check text linter rules
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      # Make sure that we only install the dependencies for textlint to speed up install
+      - run: |
+          jq 'del(.dependencies) | .scripts |= with_entries(select(.key | contains("check:"))) | .devDependencies |= with_entries( select(.key | startswith("textlint")))' package.json > package2.json && mv package2.json package.json
```

Comment:
A few questions:

- Why prune the scripts?
- Simplify 
  `package.json > package2.json && mv package2.json package.json` 
   to just 
   `package.json > package.json`?

Reply:
* if I don't prune the scripts it runs all of the npm install dependencies which loads submodules, etc., which is a big hit on performance
* because if I do jq package.json > package.json the input file is overwritten and I get an empty file

Reply:
- For scripts, all you should need is to removed `prepare` -- I don't know if that can help simplify the jq arguments.
- Ok (re file override)

Reply:
Oh, actually, without modifying `scripts`, you can avoid `prepare` being run by adding the `--ignore-scripts` flag to the two `npm` commands below.